Win32.cabal

```diff
@@ -1,5 +1,5 @@
 name:           Win32
-version:        2.13.2.0
+version:        3.0.0.0
```

Comment:
For the final version, could you revert this.  A bump in the version number triggers the CI to automatically release the package.